ts/packages/ts/src/commands/commands.test.ts

```diff
@@ -377,24 +377,49 @@ describe(`commands`, () => {
     });
   });
   describe(`ExportCommand`, () => {
```

Comment:
NIT: Depending on your input regarding `filter` overriding `hidden` filtering, maybe we should add a test that covers a scenario where both situations occur at the same time to ensure we don't miss this situation in the future.